ffmpeg/ffmpeg_errors.go

```diff
@@ -49,6 +49,34 @@ func error_map() map[int]error {
 
 var ErrorMap = error_map()
 
+func non_retriable_errs() []string {
+	errs := make([]string, 0, 20)
```

Comment:
Maybe we can use a nil slice that is appended to so we don't have to worry about a hardcoded capacity? Incurs additional allocations under the hood, but probably ok since this function is called once to set the exported package variable.